reesegreen2014/TaskPro
# Feature/about page

#### What does this PR do?
This PR implements several new features and improvements:
- Adds an About page with team member cards and links to individual reflections.
- Implements routing for navigating to different pages, including the home page, about page, and individual reflections.
- Adds a 404 Not Found page with a humorous message and a link back to the homepage.
- Updates the styling and ensures consistency across different components.
#### Where should the reviewer start?
- Review the new routing setup in `App.js`.
- Check the new `AboutPage.js` and `AboutPage.css` for the team member cards and links to reflections.
- Review the `NotFound.js` and `NotFound.css` for the 404 page implementation.
#### How should this be manually tested?
- Start the application and navigate to different routes:
- `/` for the homepage.
- `/about` for the about page.
- `/reflections/member1`, `/reflections/member2`, `/reflections/member3` for individual reflections (Kevin and Jarvis to add later, Reese to edit draft)
- Navigate to a non-existent route to see the 404 Not Found page.
- Check the layout and styling on different screen sizes to ensure responsiveness.
- Verify that the links on the about page correctly navigate to the reflections and that the back button on the reflections page works as expected.
#### Any background context you want to provide?
- These changes are part of the ongoing improvements to enhance the user experience and navigation within the TaskPro application. The 404 page was added to handle invalid routes gracefully, and the About page provides information about the team and their reflections on the project.
#### Screenshots (if appropriate):
![image](https://github.com/user-attachments/assets/4f36affa-af62-4669-98c7-51a1206ee985)
![image](https://github.com/user-attachments/assets/193e2c0f-0a0a-468c-b115-df66398727b7)
